app/views/bookmarks/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<% # Overriden to remove duplicate id of "content" %>
```

Comment:
It's helpful to also have a link to the upstream file / line that we've overridden.